src/lib/source-handlers/bitbucket-server/project-is-empty.ts

```diff
@@ -0,0 +1,21 @@
+import { getBitbucketServerToken } from './get-bitbucket-server-token';
+import { fetchAllRepos } from './list-repos';
+import { BitbucketServerRepoData } from './types';
+
+export async function bitbucketServerProjectIsEmpty(
+  projectKey: string,
+  sourceUrl?: string,
+): Promise<boolean> {
+  const bitbucketServerToken = getBitbucketServerToken();
+  if (!sourceUrl) {
+    throw new Error(
+      'Please provide required `sourceUrl` for Bitbucket Server source',
+    );
+  }
+  const repos = (await fetchAllRepos(
+    sourceUrl,
+    projectKey,
+    bitbucketServerToken,
+  )) as BitbucketServerRepoData[];
+  return (!repos || repos.length === 0) ? true : false;
```

Comment:
`return (!repos || repos.length === 0)` is already a boolean there is no need for anything else.

Reply:
Done